agimus_demo_05_pick_and_place/agimus_demo_05_pick_and_place/orchestrator.py

```diff
@@ -205,7 +205,7 @@ def go_to(self, desired_configuration):
             list(current_robot_state.position), desired_configuration
         )
         self.publish(traj)
-        self.hpp_client.restart()
+        # self.hpp_client.restart()
```

Comment:
If you have a particular motivation for commenting this, can you pls add a comment?

Reply:
will do. It is commented since it does not work properly atm (corba crashes)